definitions/variable/industry/production.yaml

```
@@ -0,0 +1,56 @@
+- Production|Iron and Steel|{Iron Commodity}:
+    description: Production of {Iron Commodity}
+    unit: Mt
+    tier: 1
```

Always useful to include a cross-reference if the variables were renamed compared to earlier projects
```suggestion
    tier: 1
    navigate: Production|Iron and Steel|{Iron Commodity}|Volumne
```